### ISSUE TITLE
Selective offline storage

### ISSUE THREAD

**Issue by di72nn** [wallabag/android-app#361]
Allow to select specific categories/tags to store for offline reading (don't store other articles).

**Comment by anarcat** [comment]
one thing i would expect from this would be for the software to download only the articles i "swipe". you can take the Materialistic app (hackernews android app) for example: articles are not downloaded by default, unless you swipe them.

**Comment by Strubbl** [comment]
I do not get, what "swiping an article" means

**Comment by anarcat** [comment]
> I do not get, what "swiping an article" means

in Materialistic, an article is a block of text with controls in a list of articles. if i drag my finger across the article block, it moves to the side and shows "save", if i remove my finger, the article is saved. if i drag it back, it's not. you really got to try it to see. :)

**Comment by Strubbl** [comment]
Okay, so by default, you would only have articles offline which you marked with a finger? That's a tough job if you have many articles. I think i would not use this option and save all by default. The risk of forgetting to swipe before reading offline is too high.

**Comment by di72nn** [comment]
(If I understand the Materialistic app nature correctly) Unlike the Materialistic app, wallabag is populated by the user himself, so the user is likely to be interested in the most of articles. But I think the ability to manually select articles one-by-one should be considered too.

**Comment by anarcat** [comment]
yes, that is a good distinction. i use wallabag as a bookmarking software though, so there's a lot of stuff i file that i don't actually need on the device... at least read stuff shouldn't be downloaded...

**Comment by di72nn** [comment]
I believe you're not alone in using wallabag for bookmarking, so multiple options should be considered.

// I switched to [Shaarli](https://github.com/shaarli/Shaarli) for bookmarking (wallabag is left for saving content and reading).

**Comment by anarcat** [comment]
On 2016-12-01 03:18:49, Dmitriy Bogdanov wrote:
> I believe you're not alone in using wallabag for bookmarking, so multiple options should be considered.
>
> // I switched to [Shaarli](https://github.com/shaarli/Shaarli) for bookmarking (wallabag is left for saving content and reading).

i'd rather avoid having yet another application to do basically the same
thing. i don't understand why people use both shaarli *and* wallabag,
but that's offtopic here. :)
